server/ServerFSM.cpp

```diff
@@ -3349,14 +3414,14 @@ sc::result ProcessingTurn::react(const ProcessTurn& u) {
     server.PostCombatProcessTurns();
 
     // update players that other empires are now playing their turn
-    for (const auto& empire : server.Empires()) {
+    for (auto& empire : server.Empires()) {
```

Comment:
shouldnt that still be `const auto& empire` ?
Ready() has a const function body.

Reply:
Fixed